translate/lang/test_ro.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+
+from translate.lang.ro import RomanianChecker
+
+
+def test_cedillas():
+    """Tests that we can detect cedillas."""
+    language = RomanianChecker()
+    assert language.cedillas(u"",u"blaŢbla") == True
+    assert language.cedillas(u"",u"blaŞbla") == True
+    assert language.cedillas(u"",u"blaţbla") == True
+    assert language.cedillas(u"",u"blaşbla") == True
+    assert language.cedillas(u"",u"blaŢblaŞblaţblaşbla']") == True
+    assert language.cedillas(u"",u"blașțăâîȘȚĂÂÎbla") == False
+    assert language.cedillas(u"",u"abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ") == False
```

Comment:
Use `is` instead of `==` to compare against booleans. Change all the occurrences in the test.